src/types.tsx

```diff
@@ -890,6 +926,207 @@ export interface SearchBarProps {
+   * Only available from iOS 26.0 and later.
+   *
+   * Read more: https://developer.apple.com/documentation/uikit/uibarbuttonitem/identifier
+   */
+  identifier?: string;
+  /**
+   * A badge to be rendered on a bar button item.
+   * Only available from iOS 26.0 and later.
+   *
+   * Read more: https://developer.apple.com/documentation/uikit/uibarbuttonitembadge
+   */
+  badge?: {
+    /**
+     * The text to display in the badge.
+     */
+    value: string;
+    /**
+     * The color of the badge.
+     */
+    color?: ProcessedColorValue;
+    /**
+     * The background color of the badge.
+     */
+    backgroundColor?: ProcessedColorValue;
+    style?: {
+      fontFamily?: string;
+      fontSize?: number;
+      fontWeight?: string;
+    };
+  };
+  accessibilityLabel?: string;
+  accessibilityHint?: string;
+}
+
+export interface HeaderBarButtonItemWithAction
+  extends SharedHeaderBarButtonItem {
+  /**
+   * A unique identifier for the button item.
+   * This is required to identify which menu item was selected in `onPress`.
+   */
+  buttonId: string;
+  /**
+   * A Boolean value that indicates whether the button is in a selected state.
+   *
+   * Read more: https://developer.apple.com/documentation/uikit/uibarbuttonitem/isselected
+   */
+  selected?: boolean;
+  /**
+   * A Boolean value that indicates whether the button represents an action or selection.
+   * Only available from iOS 15.0 and later.
+   *
+   * Read more: https://developer.apple.com/documentation/uikit/uibarbuttonitem/changesselectionasprimaryaction
+   */
+  changesSelectionAsPrimaryAction?: boolean;
+}
+
+export interface HeaderBarButtonItemMenuAction {
+  title?: string;
+  type: 'action';
+  /**
+   * A unique identifier for the menu item.
+   * This is required to identify which menu item was selected in `onPress`.
+   */
+  menuId: string;
+  /**
+   * Any SF symbol. Explore them here: https://developer.apple.com/sf-symbols/
+   */
+  sfSymbolName?: string;
+  /**
+   * State of the menu item.
+   *
+   * Read more: https://developer.apple.com/documentation/uikit/uimenuelement/state
+   */
+  state?: 'on' | 'off' | 'mixed';
+  /**
+   * Style of the menu item.
+   *
+   * Read more: https://developer.apple.com/documentation/uikit/uimenuelement/attributes
+   */
+  attributes?: 'destructive' | 'disabled' | 'hidden' | 'keepsMenuPresented';
+  /**
+   * Discoverability title of the menu item.
+   *
+   * Read more: https://developer.apple.com/documentation/uikit/uiaction/discoverabilitytitle
+   */
+  discoverabilityTitle?: string;
+}
+
+export interface HeaderBarButtonItemSubmenu {
+  title?: string;
+  type: 'submenu';
+  /**
+   * Any SF symbol. Explore them here: https://developer.apple.com/sf-symbols/
+   */
+  sfSymbolName?: string;
+  items: HeaderBarButtonItemWithMenu['menu']['items'];
+}
+
+export interface HeaderBarButtonItemWithMenu extends SharedHeaderBarButtonItem {
+  menu: {
+    title?: string;
```

Comment:
@johankasperi We discovered [this crash](https://github.com/mpiannucci/react-native-context-menu-view/issues/147) when running our app on Mac OS. So, the title prop on ContextMenu is required on Mac OS. Not sure if it makes sense to change the type here accordingly?

Reply:
I'd recommend against it. MacOS is not a supported platform. If you get it running any way - that's great, but we should not take it into consideration in API. 

We might leave a comment however.

Reply:
> Just to be clear, it's iPadOS running on a Silicon Mac 

Mac Catalyst? Okay, I won't oppose here. But if anything I'd go with what @joluet suggested here: 

> Edit: Setting the title to an empty string fixes the crash and hides the title. So I think the type can stay like this but in the native code it might make sense to default to an empty string for the menu title.